<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEF86_64</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-science">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;QEF86_64&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A Simple and minamal GUI frontend for qemu-system-x86_64</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Machines</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>177</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch Virtual Machine</string>
     </property>
    </widget>
    <widget class="QLabel" name="vm_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>VM name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CD ROM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>361</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HARD DRIVE</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="create_new_vm">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New VM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Update VM List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
